### PR TITLE
Added preview animation in readme.md (#3)

### PR DESCRIPTION
**Pull request by pradipta007**
Added the given animation just after features section with alt text.

### CODE REVIEW

**Comment by pradipta007** [comment]
#8 @SgiobairOg Hi i have made the required changes :)

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Color Vision is a utility extension for VSCode that provides a way of visualizin
 
 Color Vision detects HEX, HEXA, RGB, and RGBA and displays a color swatch next to the color value so that you can see exactly how the color will look.
 
+![alt text](resources/images/usage_15102019_512.gif "Example usage of the Color Vision extension showing color swatches being added to HEX and RGB color definitions")
```
Comment (SgiobairOg) [inline README.md]:
```suggestion
![Example usage of the Color Vision extension showing color swatches being added to HEX and RGB color definitions](resources/images/usage_15102019_512.gif)
```

**Review by SgiobairOg — CHANGES_REQUESTED** [review]
Looks good, just need to move the text description. The quoted section after the url renders as a `title` attribute, the text in the square brackets is what renders as `alt` text.

File: README.md (hunk 1 above)
Comment (pradipta007) [reply README.md]: was my updated readme.md file taken?

**Review by SgiobairOg — APPROVED** [review]
All good, thanks for contributing!

**Pull request merged**